pages/content/amp-dev/documentation/guides-and-tutorials/start/bento_guide.md

```diff
@@ -0,0 +1,295 @@
+[/tip]
+
+# Use Bento components in non-AMP pages
+
+Bento components are still in development and APIs might change. However, they're available for experimental use in a developer preview. This means you must enable the experimental flag either in the document or the console. This is no different from other [experiments in AMP](../learn/experimental.md). 
+
+Keep an eye out for announcements and updates. While Bento AMP is in developer preview, the team will continue to make necessary changes to the API to improve features.
+
+
+## Enable Bento Experiment 
+
+Bento components are available experimentally and you must [enable experimental features](../learn/experimental.md).
+
+Enable the Bento experiment by including the script below:
+
+
+```
+<script>
+  (self.AMP = self.AMP || []).push(function (AMP) {
+    AMP.toggleExperiment('bento', true);
+  });
+</script>
+```
+
+## Import AMP runtime and Bento component script
+
+You must include the AMP runtime script and import the component script for each individual Bento component desired. We also recommend importing the component's CSS stylesheet for the best user experience.
+
+```
+<!-- AMP runtime script -->
+<script async src="https://cdn.ampproject.org/v0.js"></script>
+<!-- Bento component stylesheet -->
+<link rel="stylesheet" type="text/css" href="https://cdn.ampproject.org/v0/amp-bento-component-name-1.0.css">
+<!-- Bento component script -->
+<script async custom-element="amp-bento-component-name" src="https://cdn.ampproject.org/v0/amp-bento-component-name-1.0.js"></script>
+``` 
+
+[tip type="note"]
+To make Bento components load event faster, eliminating need for the AMP runtime script is a high priority for Bento AMP. Follow our progress on the [Bento roadmap](../../../community/roadmap.html?category=bento#status-updates).
+[/tip]
+
+Read each Bento component’s reference documentation for implementation details.
+
+## Layout and styling
+
+Each Bento component has a small CSS library you must include to guarantee proper loading without [content shifts](https://web.dev/cls/). Because of order-based specificity, you must manually ensure that stylesheets are included before any custom styles.
+
+```
+<link rel="stylesheet" type="text/css" href="https://cdn.ampproject.org/v0/amp-bento-component-name-1.0.css">
+<style>
+ custom styles
+</style> 
+```
+
+Include additional standard CSS to further define layouts and styles. Here are a few strategies on how you can layout Bento components using CSS:
+
+### Fixed layout
+
+```css
+.fixed-layout {
+    display: inline-block;
+    width: xpx;
+    height: xpx;
+}
+```
+
+### Container layout
+
+```css
+.container-layout {
+...
+}
+```
+
+### Responsive layout with a fixed aspect ratio
+
+```css
+.responsive-layout {
+    display: block;
+    width: 100%;
+    height: auto;
+    aspect-ratio: width / height;
+}
+```
+
+### Fill layout
+
+```css
+.fill-layout {
+...
```

Comment:
This is either:

```
position: absolute;
top: 0;
left: 0;
right: 0;
bottom: 0;
```

or
```
width: 100%;
height: 100%;
```

Depending on the context. The first one is fairly universal.